launch_ros/launch_ros/descriptions/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""descriptions Module."""
+
+from .composable_node import ComposableNode
+
+__all__ = [
+    'ComposableNode',
```

Comment:
@wjwwood mentioned making this `launch_ros.ComposableNode` instead of having a descriptions module. https://github.com/ros2/launch/pull/173#discussion_r253234298

I'm fine with either if @wjwwood is on board.

Reply:
Oh, I see. The stubs did not match the examples so I chose to go with the later. But I can back up.

Reply:
I don't have a strong opinion about it, I'll leave it up to you guys based on what you think and my previous comment.